flake.nix

```diff
@@ -0,0 +1,23 @@
+{
+  description = "Nix Flake for kf-website";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      with import nixpkgs { inherit system; };
+      let project = pkgs.callPackage ./yarn-project.nix { } { src = ./.; };
+      in {
+
+        defaultPackage = project.overrideAttrs (oldAttrs: {
+          buildInputs = oldAttrs.buildInputs ++ [ pkgs.python3 ];
+          buildPhase = "yarn build";
+          installPhase = "mv public $out";
+        });
```

Comment:
I think you might not need to do any of this. I think `project` is already your built app.

Reply:
But I haven't tested this!

Reply:
yarn-plugin-nixify does not appear to create `defauilt.nix` when `flake.nix` is present in the same directory, and then it also differs from what you see above: here we execute build step and copy only the static files to the output directory